src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -758,9 +758,20 @@ static int rbd_nbd(int argc, const char *argv[])
 
   argv_to_vec(argc, argv, args);
   env_to_vec(args);
-  auto cct = global_init(NULL, args, CEPH_ENTITY_TYPE_CLIENT,
-			 CODE_ENVIRONMENT_DAEMON,
-			 CINIT_FLAG_UNPRIVILEGED_DAEMON_DEFAULTS);
+
+  code_environment_t code_env = CODE_ENVIRONMENT_UTILITY;
+  int env_flag = 0;
+  for (auto arg : args) {
+    if (arg != NULL && arg[0] != '-') {
+      if (strcmp(arg, "map") == 0) {
+        code_env = CODE_ENVIRONMENT_DAEMON;
+        env_flag = CINIT_FLAG_UNPRIVILEGED_DAEMON_DEFAULTS;
+      }
+      break;
```

Comment:
ah... yes... seems we must support both --device=/dev/nbd0 and --device /dev/nbd0.

In this way, we cannot pre-scan the argument vector and check the first argument that doesn't start w/ a "-". I should make a pre-scan, and strcmp each argument with "map". 

@trociny @dillaman , do you agree?